components/Header/HamburgerLink.tsx

```diff
@@ -0,0 +1,21 @@
+import { Link } from "react-router-dom";
+import { send } from "../../stateSignal";
+import styles from "./Header.module.scss";
+
+interface LinkProps {
+    location: string;
+    name: string;
+}
+
+export default function HamburgerLink({ location, name }:LinkProps) {
+    const is_current_location = window.location.pathname === location;
```

Comment:
window?, component may not have access to window, only in useEffect and useLayoutEffect

Reply:
[window.location.pathname]